dataikuapi/dss/wiki.py

```diff
@@ -89,6 +89,30 @@ def create_article(self, article_name, parent_id=None, content=None):
 
         return article
 
+    def get_export_stream(self, paperSize="A4", exportAttachment=False):
+        """
+        Download the whole wiki of the project in PDF format as a binary stream.
+        Warning: this stream will monopolize the DSSClient until closed.
+        """
+        body = {
+            "paperSize": paperSize,
+            "exportAttachment": exportAttachment
+        }
+        return self.client._perform_raw("POST", "/projects/%s/wiki/actions/export" % (self.project_key), body=body)
+
+    def export_to_file(self, path, paperSize="A4", exportAttachment=False):
+        """
+        Download the whole wiki of the project in PDF format into the given output file.
+        """
+        stream = self.get_export_stream(paperSize=paperSize, exportAttachment=exportAttachment)
```

Comment:
is it possible to use `with ... as stream:` instead of the get/do/close construction? (if the stream can handle it)

Reply:
I added it, it's indeed neater. Also I used underscores instead of camelCase for parameter names